specification/monitor/resource-manager/Microsoft.Insights/preview/2021-02-01-preview/examples/listOperations.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.Storage/scheduledQueryRules/write",
```

Comment:
>"name": "Microsoft.Storage/scheduledQueryRules/write", [](start = 12, length = 54)

Examples should contain all the operations as these are used for documentation generation also. Please add other operations as well.

Reply:
Done.